source/dofs/dof_tools_constraints.cc

```diff
@@ -1288,20 +1288,40 @@ namespace DoFTools
                         // 2) there is no dominating FE among sub faces (e.g.
                         // Q1xQ2 vs Q2xQ1), but subfaces still dominate mother
                         // face (e.g. Q2xQ2). To cover this case we would have
-                        // to use find_least_dominating_fe_in_collection()
-                        // of FECollection with fe_indices of sub faces. 3)
-                        // Finally, it could happen that we got here because
+                        // to find the least dominating element amongst all
+                        // finite elements on sub faces. 3) Finally, it could
```

Comment:
```suggestion
                        // finite elements on sub faces. 
                        //
                        // 3) Finally, it could
```